modules/azure-public-dns/dns-records.auto.tfvars

```diff
@@ -0,0 +1,36 @@
+project   = "hmcts"
+component = "public-dns"
+location  = "uksouth"
+env       = "mgmt"
+
+recordsets = [
```

Comment:
are these loaded or just in the repo as an example?

Reply:
These are just an example

Reply:
probably this file should be removed if it is for example purpose.  tfvar file shouldn't go under the module.

Reply:
agreed same as the zones comment: https://github.com/hmcts/azure-platform-terraform/pull/5/files#r334818087